state-chain/chains/src/sol/sol_tx_building_blocks.rs

```diff
@@ -1084,7 +1088,7 @@ mod tests {
 			let transaction_bytes = serialized_tx.len();
 			println!(
 				"Full rotation - number of nonce accounts: {} - Bytes: {}",
-				i,
+				i + 1,
```

Comment:
In these tests, what are the println! statements for?
Should we be using assertions instead? If the prinlns are not being used I would rather we remove them, otherwise it just creates noise.

Reply:
In general those prints are for development purposes and I haven't removed them because I'll most likely need them later on when testing and/or playing with some transaction parameters. But it makes sense to remove them and I'll just add them locally when needed.